processYear.ts

```diff
@@ -193,6 +187,7 @@ export const processClustersForYear = async (
       results.numberOfClusters = results.clusterNumbers.length;
       console.log(lcaTotals);
 
+      const KM_TO_MILES = 0.621371;
```

Comment:
why remove this from the transportation export but also re-define it inside that file?  It would seem more useful to keep exporting it in case something needs the conversion factor too.

Reply:
I thought exporting such a conversion factor was unnecessary and might not be considered a good practice. I will change it back.